tests/integration/helpers.py

```diff
@@ -1135,6 +1134,8 @@ async def backup_operations(
         **credentials,
     )
     await action.wait()
+
+    await ops_test.model.relate(database_app_name, s3_integrator_app_name)
```

Comment:
for HTTPS communication on pgbackrest to work, it first needs the certs passed on through s3-integrator config, hence the relocation of this relation to after the `set_config` command.